CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Change history for stripes-cli
 
-## IN PROGRESS
+## 1.30.0 (IN PROGRESS)
```

Comment:
@zbure did you mean `1.13.0` rather than `1.30.0`?

Reply:
I absolutely meant `1.13.0`.